scripts/install-argocd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
```

Comment:
@dewan-ahmed I have removed few script that was doing local installation check. Let do the installation for CI first, later we can leverage it to local installation. The script looks good to me, however you missed one step of converting the Upper case to lower case. And no need to put the binary in the default linux path because this script need to be be sourced after ```make bin``` - https://github.com/redhat-developer/kam/blob/master/scripts/openshiftci-presubmit-all-tests.sh#L10. 
```
# Find the latest ArgoCD version
VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
echo $VERSION

# Platform check
PLATFORM=`uname -s | awk '{print tolower($0)}'`
echo $PLATFORM 

# Fetch the appropriate OS binary for ArgoCD
curl -sSL -o argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-$PLATFORM-amd64

# Make the argocd CLI executable
chmod +x argocd 
```

Reply:
@amitkrout thanks for the feedback and fix. I implemented your feedback and tested on darwin & linux.